Docs/ProjectSpec.md

```diff
@@ -404,6 +404,23 @@ targets:
     type: framework
 ```
 
+**SDK Dependency**
+
+- [ ] **embed**: **String** - Root of framework path. Default value is `BUILT_PRODUCTS_DIR`
```

Comment:
```suggestion
- [ ] **root**: **String** - Root of framework path. Default value is `BUILT_PRODUCTS_DIR`
```

Reply:
I am sorry 🙈 I fixed it in commit d23271a073c736a85583840fef39543e51ae6d6b.

Reply:
Perhaps we can also mention that a common possible value for this is `DEVELOPER_DIR`. Are there other common cases?

Reply:
@raptorxcz

Reply:
Available value are:

```
    case none
    case absolute
    case group
    case sourceRoot
    case buildProductsDir
    case sdkRoot
    case developerDir
    case custom(String)
```

We can allow user to set any of the value without `custom` and map it directly to this enum, or we can let user to set any value he wants and just add more info to ProjectSpec.md file. What would you prefer?

Reply:
I think keeping it as custom is fine. But maybe we can just add:
```
Root of framework path, for example `DEVELOPER_DIR`. Default value is `BUILT_PRODUCTS_DIR`
```